src/tool/hpcrun/gpu/nvidia/cupti-api.c

```diff
@@ -722,53 +717,17 @@ cupti_load_callback_cuda
  size_t cubin_size
 )
 {
-  // Compute hash for cubin and store it into a map
-  cubin_hash_map_entry_t *entry = cubin_hash_map_lookup(cubin_id);
-  unsigned char *hash;
-  unsigned int hash_len;
-  if (entry == NULL) {
-    cubin_hash_map_insert(cubin_id, cubin, cubin_size);
-    entry = cubin_hash_map_lookup(cubin_id);
-  }
-  hash = cubin_hash_map_entry_hash_get(entry, &hash_len);
-
-  // Create file name
-  char file_name[PATH_MAX];
-  char hash_string[PATH_MAX];
-  size_t used = 0;
-  size_t i;
-  for (i = 0; i < hash_len; ++i) {
-    used += sprintf(&hash_string[used], "%02x", hash[i]);
-  }
+  uint32_t loadmap_module_id;
+  bool mark_used = false;
```

Comment:
(Same as previous comment).
This constant does not need to be a variable. Using `true /* mark_used */` in the argument list is equivalently readable. (At the very least make it `const`).

Reply:
done.